open-metadata-implementation/access-services/data-engine/data-engine-server/src/main/java/org/odpi/openmetadata/accessservices/dataengine/server/admin/DataEngineServicesInstance.java

```diff
@@ -42,7 +42,9 @@
     DataEngineServicesInstance(OMRSRepositoryConnector repositoryConnector, List<String> supportedZones, List<String> defaultZones,
                                OMRSAuditLog auditLog, String localServerUserId, int maxPageSize) throws NewInstanceException {
 
-        super(description.getAccessServiceName(), repositoryConnector, supportedZones, defaultZones, auditLog, localServerUserId, maxPageSize);
+
+        super(description.getAccessServiceName() + " OMAS", repositoryConnector, supportedZones, defaultZones, auditLog,
```

Comment:
@popa-raluca Hi Raluca, I just tested this change with POST call createExternalDataEngine and it will throw a 404 exception. If you remove the " OMAS" and it will work. Not sure the the impact on the asset scan from @mandy-chessell but just let you know about it.